modules/config/src/main/scala/scala/cli/config/Key.scala

```diff
@@ -79,7 +81,8 @@ object Key {
   final class StringEntry(
     val prefix: Seq[String],
     val name: String,
-    val description: String = ""
+    val description: String = "",
+    override val hidden: Boolean = false
```

Comment:
Is this necessary since the default value for `hidden` is false? The same example below.

Reply:
Maybe not strictly necessary, but IMO it's useful.
It allows to not have to strictly pass the value to the constructor, if a key isn't to be hidden.
And at the same time, the default in `Key` is defined so that it's  inherited for keys initialized via anonymous classes (and has to be overridden there only if a key is hidden, like i.e. `Keys.repositoryCredentials`).